### DISCUSSION TITLE
getaddrinfo ENOTFOUND localhost     at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:internal/dns/promises:82:17)

### DISCUSSION THREAD

**Discussion by elarbor** [vuejs/vitepress#3725] [Q&A] [answered]
![image](https://github.com/vuejs/vitepress/assets/43592730/039b725b-b1d3-463f-a6fe-a006650f7bd3)
![image](https://github.com/vuejs/vitepress/assets/43592730/5d476863-9431-4eee-9cac-0c0e29e2d17a)
![image](https://github.com/vuejs/vitepress/assets/43592730/a7f5b093-18aa-45c6-86f4-8d074bffbc96)
是哪里不对吗？

**Comment by brc-dd** [comment, marked as answer]
Not sure. I'm on node 21.7.1 and it's working fine. Maybe try specifying host manually:

```ts
// .vitepress/config.ts

import { defineConfig } from 'vitepress'

export default defineConfig({
  vite: {
    server: {
      host: '127.0.0.1'
    }
  }
})
```

**Reply by elarbor** [reply]
果然有用，感谢
![image](https://github.com/vuejs/vitepress/assets/43592730/e90a744a-79e6-44ed-b805-7022a70e3fc0)
不过这是为什么呢？

**Reply by brc-dd** [reply]
Not sure. Can you update your `/etc/hosts` file to this and check if it's working without manually specifying vite.server.host?

```hosts
127.0.0.1	localhost
255.255.255.255	broadcasthost
::1             localhost
```

**Reply by elarbor** [reply]
加上了 `::1             localhost` 不管用。算了吧，哈哈哈哈哈，放弃了。反正配置`vite.server.host`也管用
